wdl/transforms/biscayne/src/main/scala/wdl/transforms/biscayne/ast2wdlom/BiscayneGenericAstNode.scala

```diff
@@ -0,0 +1,34 @@
+package wdl.transforms.biscayne.ast2wdlom
```

Comment:
Could we use type parameters to reduce duplication between this file and `Draft3GenericAstNode.scala`? The only difference seems to be the type of the `Ast` (`wdl.draft3.parser.WdlParser.Ast` vs. `wdl.biscayne.parser.WdlParser.Ast`).

Incomplete but illustrates what I'm getting at:
```
case class BaseGenericAst[T <: GenericAstNode](ast: T) extends GenericAst {
  override def getAttribute(attr: String): GenericAstNode = Option(ast.getAttribute(attr)).map(T.apply).orNull
  override def getAttributes: Map[String, GenericAstNode] = ast.getAttributes.asScala.toMap collect {
    case (key, value) if value != null => key -> T(value)
  }
  override def getName: String = ast.getName
}
```